framework/test/gtest/BundleActivatorTest.cpp

```diff
@@ -27,39 +27,67 @@
 #include "cppmicroservices/FrameworkEvent.h"
 #include "cppmicroservices/FrameworkFactory.h"
 
+#include "TestUtilBundleListener.h"
 #include "TestUtils.h"
-#include "TestingConfig.h"
-#include "TestingMacros.h"
+
+#include "gtest/gtest.h"
 
 using namespace cppmicroservices;
 
+class BundleActivatorTest : public ::testing::Test
```

Comment:
Take a look at https://github.com/google/googletest/blob/master/googletest/docs/primer.md#test-fixtures-using-the-same-data-configuration-for-multiple-tests-same-data-multiple-tests for guidelines on how to define the test fixture.